### DISCUSSION TITLE
Shortcut for 'jump to beginning' menu

### DISCUSSION THREAD

**Discussion by macosrom** [iina/iina#4784] [Ideas]
Thanks for great app!

I was seeking for a way to  use 'jump to beginning' function in Shortcut cause IINA had a menu for it, but there wasn't a shortcut.

There might be some reason, like in the middle of watching the movie, people may unintentionally press the key and it might be annoying thing for them. 

But people like me, using IINA for practicing guitar play with good example and want to keep going back to beginning.
I know that there is loop function, but I need set it every time by two click.

so, I hope that IINA can have complex shortcut for 'jump to beginning' by default or by optional in preference and shortcut needs to customizable.

How does sound?

**Comment by lhc70000** [comment]
Hi, you should be able to add this shortcut by setting the action to `Seek to 0 s`.
(There's a UI glitch in the screenshot below, it should have a 0 in the text box. Anyway make sure the corresponding mpv command is `seek 0 absolute`)

<img width="968" alt="image" src="https://github.com/iina/iina/assets/8478049/dfed795e-29cf-4e63-b259-fa3c1a772053">

**Reply by macosrom** [reply]
Thanks a lot, by the way, Is it required to add a set of new shortcut?
